embedded/ios/Package.swift

```diff
@@ -0,0 +1,18 @@
+// swift-tools-version: 6.0
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "EmbeddedElementCall",
```

Comment:
I would add a platform and version specification below here
` platforms: [.iOS(.v17_6)],`

This sets the minimum iOS version that should build, and I think it should match EL-X which currently supports 17.6 as the minimum supported iOS version.
MacOS min versions can be inferred by the iOS version. But if you want to add it to match EL-X, the minimum is 14.6

Reply:
```suggestion
    name: "EmbeddedElementCall",
    platforms: [.iOS(.v17_6)],
```

Reply:
Actually just using `iOS v17` should be sufficient, macOS isn inferred, and is easier to test on older sim versions since 17.6 is not provided as a valid sim by Apple